modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/modals/constants.ts

```diff
@@ -3,12 +3,13 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
```

Comment:
To me, all these modal utils belong to the `modal` folder, as they are directly related. The same occurs with other utils folders.

Reply:
Yeah, makes sense to me. Root `utils` dir seems to be best reserved for general-purpose utils, used in multiple components. Restructuring now is probably not worth the effort, as we want to migrate away to `openModal`, see https://github.com/liferay-frontend/liferay-portal/pull/3781#discussion_r1373153346, which will probably result in us deleting all these utils.